networkx/linalg/algebraicconnectivity.py

```diff
@@ -377,6 +377,21 @@ def algebraic_connectivity(
     See Also
     --------
     laplacian_matrix
+
+    Examples
+    --------
+    For undirected graphs algebraic connectivity can tell us if a graph is connected or not
+    G is connected iff  :math: `algebraic\_connectivity(G)>0`
+
+    >>> #if G is a complete grpah then G is connected
+    >>> G = nx.complete_graph(5)
+    >>> nx.algebraic_connectivity(G) > 0
+    True
+    >>> #Add a node so G is no longer connected
+    >>> G.add_node(10)
+    >>> nx.algebraic_connectivity(G) > 0
+    False
```

Comment:
Just a couple more formatting nits - moving/deleting comments to try to simplify things. Not blockers just suggestions!

```suggestion
    >>> G = nx.complete_graph(5)
    >>> nx.algebraic_connectivity(G) > 0
    True
    >>> G.add_node(10)  # G is no longer connected
    >>> nx.algebraic_connectivity(G) > 0
    False
```

Reply:
Yes, the suggestions are great! I'm learning a lot. Thanks for the review @rossbar!